common/ap_exams.ts

```diff
@@ -0,0 +1,546 @@
+import { TransferableExamGroup } from "./types";
+
+export const APExamGroups2020To2021: Array<TransferableExamGroup> = [
```

Comment:
AP exams as listed [here](https://www.northeastern.edu/admissions/wp-content/uploads/2020/05/AP-2020-2021.pdf)

Reply:
I see you're an `Array<T>` and not a `T[]` kinda guy...